src/test/java/com/adyen/BinLookupTest.java

```diff
@@ -0,0 +1,93 @@
+/*
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Java API Library
+ *
+ * Copyright (c) 2019 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ */
+package com.adyen;
+
+import org.junit.Test;
+import com.adyen.model.Amount;
+import com.adyen.model.PaymentRequest;
+import com.adyen.model.binlookup.CostEstimateAssumptions;
+import com.adyen.model.binlookup.CostEstimateRequest;
+import com.adyen.model.binlookup.CostEstimateResponse;
+import com.adyen.model.binlookup.MerchantDetails;
+import com.adyen.model.binlookup.ThreeDSAvailabilityRequest;
+import com.adyen.model.binlookup.ThreeDSAvailabilityResponse;
+import com.adyen.service.BinLookup;
+import com.adyen.service.Payment;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Tests for
+ * /get3dsAvailability
+ * /getCostEstimate
+ */
+public class BinLookupTest extends BaseTest {
+
+
```

Comment:
It would be good to have some negative test scenarios as well.

Reply:
Indeed. Just added them.